package.json

```diff
@@ -13,25 +13,24 @@
     "@mui/x-date-pickers": "^6.19.4",
     "add-to-calendar-button": "^2.5.10",
     "caniuse-lite": "^1.0.30001589",
-    "chart.js": "^2.9.4",
+    "chart.js": "^4.4.3",
     "cropperjs": "^1.6.1",
     "date-fns": "^3.3.1",
     "history": "^5.3.0",
     "lodash": "^4.17.21",
     "luxon": "^3.4.4",
     "prop-types": ">=15.7.2",
-    "react": "^17.0.2",
+    "react": "^18.3.1",
     "react-big-calendar": "^1.10.3",
-    "react-brackets": "^0.4.7",
-    "react-chartjs-2": "^2.11.2",
+    "react-chartjs-2": "^5.2.0",
     "react-cropper": "^2.3.3",
     "react-csv": "^2.0.3",
     "react-device-detect": "^2.2.3",
-    "react-dom": "^17.0.2",
+    "react-dom": "^18.3.1",
     "react-dom-confetti": "^0.2.0",
     "react-dropzone": "^14.2.3",
     "react-ga4": "^2.1.0",
-    "react-icons": "^5.0.1",
+    "react-icons": "^5.2.1",
```

Comment:
Was this upgrade required for React 18?

Reply:
I looked at the changelog and it just looks like all positive things that were updated with no name changes.